website/docs/reference/appsmith-framework/widget-actions/post-message.md

```diff
@@ -114,6 +116,8 @@ When the iframe receives your message, it puts the message text into the `#targe
 
 #### From embedded page to Iframe widget
 
+![](/img/postmessage_child_outgoing.png)
```

Comment:
<img width="905" alt="Screenshot 2023-02-09 at 16 44 17" src="https://user-images.githubusercontent.com/32227135/217798110-d4f3a00f-c2e7-4302-bf6e-d6fec402cd8c.png">
I think this one can be removed for now, because the image shows communication from embedded page to Appsmith which is not possible today. The section talks about communication from element inside the Iframe widget to the the Iframe widget. so we can skip this diagram for now and add it later
```suggestion

```